tests/cli/test_cli_get.py

```diff
@@ -1,34 +0,0 @@
-from cg.store import Store
```

Comment:
why delete this?

Reply:
not deleted, it is moved to `tests/cli/get/test_cli_get.py`